### PR TITLE
Updated main loop

### PR DESCRIPTION
**Pull request by AniSanikidze**
The updated main loop, reacts to the user's input in a corresponding way.

### CODE REVIEW

File: src/chatbot/wrapper_main_loop.clj [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(defn main-loop
+  "Receives user input until a terminating keyword is met.The main loop calls help function if user input is help. Checks if the keyword is not identified and prints the random error message. Otherwise greets user or answers the questions about the park."
```
Comment (adzai) [inline src/chatbot/wrapper_main_loop.clj:2]: Add some newlines into the doc string, a line should be ~ 80 characters long

File: src/chatbot/wrapper_main_loop.clj [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  []
+  (println "Chatbot> Hi!")
+  (println "Chatbot> I am your park guide. I will tell you about Bertramka park. To end the conversation, enter 'finish'. Ask your questions")
+  (loop [user-input (read-line)]
+    (when-not (= "finish" user-input)
+      (if (= "help" user-input)
+        (help-function))
+      (if (and (= true (greeting possible-greetings user-input)) (= true (keyword-response-main user-input)))
```
Comment (adzai) [inline src/chatbot/wrapper_main_loop.clj:10]: Not quite sure what this means...

**Comment by irinakulinich3712** [comment]
Honestly, I don't quite understand why you have created this branch before the initial main-loop branch has been merged...

File: src/chatbot/wrapper_main_loop.clj (hunk 2 above)
Comment (AniSanikidze) [reply src/chatbot/wrapper_main_loop.clj:12]: If the greeting and keyword-response-main functions both return true(now, it should be changed to false), then the random error message will be printed out.

**Comment by AniSanikidze** [comment]
> Honestly, I don't quite understand why you have created this branch before the initial main-loop branch has been merged...

I created this branch after the main loop was approved and I did not think it would be a problem to create it before you merged.

File: src/chatbot/wrapper_main_loop.clj [hunk 3]
```diff
@@ -0,0 +1,14 @@
+(defn main-loop
+  "Receives user input until a terminating keyword is met.The main loop calls help function if user input is help.
+   Checks if the keyword is not identified and prints the random error message.
+   Otherwise greets user or answers the questions about the park."
+  []
+  (println "Chatbot> Hi!")
+  (println "Chatbot> I am your park guide. I will tell you about Bertramka park. To end the conversation, enter 'finish'. Ask your questions")
+  (loop [user-input (read-line)]
+    (when-not (= "finish" user-input)
```
Comment (irinakulinich3712) [inline src/chatbot/wrapper_main_loop.clj:9]: Would not it be better to use cond instead of multiple if statements?

Comment (irinakulinich3712) [reply src/chatbot/wrapper_main_loop.clj:9]: Sorry, I have highlighted wrong lines here

Comment (irinakulinich3712) [reply src/chatbot/wrapper_main_loop.clj:9]: (if (= "help" user-input)
        (help-function))
      (if (and (= false (greeting possible-greetings user-input)) (= false (keyword-response-main user-input)))
          (println (rand-nth possible-error-messages)))

Here

Comment (AniSanikidze) [reply src/chatbot/wrapper_main_loop.clj:19]: Yeah, I agree cond is better than multiple ifs. I just changed it.

**Review by adzai — APPROVED** [review]
Looks good to me, lein run works fine.

There are some formatting issues with the chatbot replies sometimes, but I would open new 
issues for it after this is merged.

**Review by irinakulinich3712 — APPROVED** [review]

**Pull request merged**